cluster/src/main/java/io/servicefabric/cluster/ClusterBuilder.java

```diff
@@ -34,6 +34,16 @@ private ClusterBuilder(ClusterEndpoint localClusterEndpoint, String wellknownMem
 		this.wellknownMembers = wellknownMembers;
 	}
 
+	public static ICluster newInstance(String memberId, int port, String seedMembers){
+		String uri = "tcp://"+memberId+"@localhost:"+port;
+		return ClusterBuilder.newInstance(ClusterEndpoint.from(uri), seedMembers).build();
+	}
+	
+	public static ICluster newInstance(String memberId, int port){
```

Comment:
currently what happens is cluster will not start
i think its wrong
cluster should try and restart